### DISCUSSION TITLE
Why can't I mut views?

### DISCUSSION THREAD

**Discussion by jgsimard** [dimforge/nalgebra#1223] [Q&A]
In the docs is says
Assignment operators do not work on any kind of view, i.e., one cannot write a *= b even if a is a mutable matrix view.
Why is that?
Will is change?
Thanks

**Comment by Andlon** [comment]
I'm actually not sure why the docs say that. This works, for example:

```rust
use nalgebra::{matrix, vector};

fn main() {
    let mut a = matrix![1.0, 2.0; 3.0, 4.0];
    let mut b = a.fixed_view_mut::<2, 1>(0, 0);
    let c = vector![5.0, 6.0];
    b += c;
}
```

Perhaps either this is not true anymore, or @sebcrozet had something else in mind? (Assuming he's the one who write this)

**Reply by sebcrozet** [reply]
Mmh, I don’t remember if that was a limitation before. The docs should be modified to say that the assignment operator `=` won’t work (instead of `*=`).

**Reply by Andlon** [reply]
Filed an issue for this: https://github.com/dimforge/nalgebra/issues/1224
